apps/src/music/progress/MusicConditions.ts

```diff
@@ -76,10 +92,17 @@ export const MusicConditions: ConditionNames = {
   },
   PLAYED_SOUND_TYPE: {
     name: 'played_sound_type',
-    valueType: 'array',
+    valueType: 'string',
     description: 'Checks if a certain sound type is playing.',
     valueOptions: ['beat', 'bass', 'lead', 'fx', 'vocal'],
   },
+  PLAYED_SOUND_TYPE_MULTIPLE_TIMES: {
+    name: 'played_sound_type_multiple_times',
+    valueType: 'string:number',
+    description:
+      'Checks if a certain sound type has played at least this many times. Ex. Value: [lead, 2]',
+    valueOptions: ['beat', 'bass', 'lead', 'fx', 'vocal'],
```

Comment:
To confirm, the `valueOptions` array will only apply to the first `valueType` part right? If later we somehow need to support multiple dropdown options, will we need to update this structure?

Reply:
Yes and yes. :)